### ISSUE TITLE
Report all shape types in Microsoft Powerpoint

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#7111]
Currently NVDA provides a role of 'shape' to any item on a Powerpoint slide that cannot be mapped directly to one of the other standard NVDA roles (E.g. table, textFrame, chart, embeddedObject etc).
However, there are over 180 shapes (including rectangles, triangles, arrows, clouds etc) that can be placed on a Powerpoint slide. Most if not all are usually used to convey particular meaning, not just for artistic reasons.
Therefore, NVDA should speak / braille these shape types, rather than just 'shape'.
To do this, we should introduce the idea of a roleText property, which can be a way for an NVDAObject to provide custom text for the role, where the role is too particular for any NVDA standard role. ARIA roleDescription can eventually use this concept as well. 
A part from these particular shapes, specific NVDA roles should be added for audio and video, as these are PowerPoint shapes that are pretty well understood concepts and would most likely be made use of outside of Powerpoint.
